code/modules/reagents/reagent_containers/jerrycan.dm

```diff
@@ -0,0 +1,178 @@
+#define LABEL_TEXT "text"
+#define LABEL_TEXT_OLD "old"
+#define LABEL_INFLAMMABLE "inflammable"
+#define LABEL_NT "nt"
+#define LABEL_NT_MINI "nt_mini"
+#define LABEL_SUSPICIOUS "sus"
+#define LABEL_SUSPICIOUS_BLACK "sus_black"
+#define LABEL_SUSPICIOUS_MINI "sus_mini"
+#define LABEL_SUSPICIOUS_MINI_BLACK "sus_mini_black"
+#define LABEL_EZ_NUTRIENT "ez"
+#define LABEL_ROBUST_HARVEST "robust"
+#define LABEL_LEFT_4_ZED "l4z"
+#define LABEL_SPACE_CLEANER "space_cleaner"
+
+#define CAP_BLACK "black"
+#define CAP_WHITE "white"
+#define CAP_RED "red"
+#define CAP_GREEN "green"
+#define CAP_BLUE "blue"
+
+	/*The jerrycan is not only a flavourful, setting appropriate way to store reagents, it offers several design benefits due to its unique combination of high roundstart capacity and bulky size.
+
+	* It prevents one coworker from hogging a shared reagent container.
+	* The high volume eases grinding and mixing without enabling big grenade creation.
+	* It allows reagents to be stolen or borrowed due to being stored outside the safety of an inventory.
+	* The highly visible inhand sprite allows you to see who is holding or transporting reagents.
+	* The smart cap flag allows you to throw the container without spilling.
+	* It reduces the tedium associated with dispensing and throwing away botany nutrient bottles, encouraging refilling.
+	* Bulky size prevent players from looting the can unless it is something they really want.
+	* Comes with a large number of visual customization options for coders who with to add new variants.*/
```

Comment:
Comments should not be indented (in this case), also if you start and end this with `/**` and `**/` on unique lines it'll both look better and also work as a doc comment (It might still work as a doc comment tbh can't recall)